### PR TITLE
#4_CreatedContactForm

### LINKED ISSUE #4: Create a Contact Form
Create a contact form with functioning php.

### PR DESCRIPTION
**Pull request by eiman-kased**
Created a form using html elements and php for validation and keeping form data after submission. 
Fixed an issue with the inline php for <textarea>. Everything is now functioning as intended.

Closes #4.

### CODE REVIEW

File: contactForm.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <title>Contact</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+</head>
+
+<body>
+    <?php
+
+    $firstName = '';
+    $lastName = '';
+    $emai = '';
```
Comment (billythakidd04) [inline contactForm.php]: `$emai`?

File: contactForm.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            </br></br>
+
+            Last Name:*<input type="text" name="lastName" id="lastName" <?= (isset($_POST['lastName']) ? 'value="' . $_POST['lastName'] . '"' : '') ?> required />
+        </div>
+        <br /><br />
+
+        <div class="phone">
+            <label for="phone">Phone Number*</br>Format:123-456-7890</label>
+            <input type="tel" id="tel" name="tel" pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}" <?= (isset($_POST['tel']) ? 'value="' . $_POST['tel'] . '"' : '') ?> />
+            </br>
```
Comment (billythakidd04) [inline contactForm.php]: whats the point of this break?

File: contactForm.php [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        </div>
+        </br></br>
+
+
+
+        <div class="email">
+            Enter an Email<input type="email" id="email" name="email" placeholder="email@example.com" <?= (isset($_POST['email']) ? 'value="' . $_POST['email'] . '"' : '') ?> required />
+            </br></br>
```
Comment (billythakidd04) [inline contactForm.php]: ?? why the breaks?

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Few problems and formatting issue

File: contactForm.php (hunk 2 above)
Comment (eiman-kased) [reply contactForm.php]: I did not like that the fields are close together. Used <br /> to add space between them.

File: contactForm.php (hunk 1 above)
Comment (eiman-kased) [reply contactForm.php]: This is why PRs are so important! Thanks!

File: contactForm.php [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+        <div class="phone">
+            <label for="phone">Phone Number*</br>Format:123-456-7890</label>
+            <input type="tel" id="tel" name="tel" pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}" <?= (isset($_POST['tel']) ? 'value="' . $_POST['tel'] . '"' : '') ?> />
+
+        </div><br />
+
+
+
+
```
Comment (billythakidd04) [inline contactForm.php]: why all the empty space? Trim down all the empty lines in the file and you're good to go.

**Review by billythakidd04 — CHANGES_REQUESTED** [review]

**Review by billythakidd04 — APPROVED** [review]

**Pull request merged**